config/locales/models/en.yml

```
@@ -10,6 +10,9 @@ en:
           attributes:
             activity:
               taken: You have already read this activity
+        gitable:
+          repository:
+            not_accessible: The repository is not accessible for our git user. Please double check if you have given the correct permissions. Note that it might take a while before a GitHub/GitLab invitation is accepted by our team.
```

```suggestion
            not_accessible: The repository is not accessible to our git user. Please double check that you have given the correct permissions. Note that it may take a while before a GitHub/GitLab invitation is accepted by our team.
```

Can we include a link to the docs here that explains the github/gitlab user thing?